new-frontend/src/environments/environment.development.ts

```diff
@@ -2,8 +2,8 @@ import { EnvironmentConfig } from 'src/app/models/application/enivronmentConfig.
 
 const env: EnvironmentConfig = {
   production: false,
-  server_url: 'http://localhost:5000',
-  api_path: '/api',
+  server_url: 'https://test-petronas.azurewebsites.net',
+  api_path: '',
```

Comment:
I'd prefer to change this back at some point - but if you want to have this right now it's fine by me

Reply:
Classic OOPS!